### PR TITLE
Feat/refactor checks to all validators remaining methods

### PR DESCRIPTION
**Pull request by IgorEulalio**

### CODE REVIEW

File: actionners/calico/networkpolicy/networkpolicy.go [hunk 1]
```diff
@@ -18,6 +18,11 @@ import (
 	"github.com/falco-talon/falco-talon/utils"
 )
 
+type Config struct {
+	Allow []string `mapstructure:"allow" validate:"omitempty"`
```
Comment (Issif) [inline actionners/calico/networkpolicy/networkpolicy.go:22]: FYI, I would like to replace it later this field by 2:
- allow_cidr
- allow_namespaces

Comment (IgorEulalio) [reply actionners/calico/networkpolicy/networkpolicy.go:22]: Ok, with this new structure would be easier to do this validation.

File: actionners/kubernetes/exec/exec.go [hunk 2]
```diff
@@ -17,6 +16,11 @@ import (
 	"github.com/falco-talon/falco-talon/utils"
 )
 
+type Config struct {
+	File  string `mapstructure:"command" validate:"required,omitempty"`
```
Comment (Issif) [inline actionners/kubernetes/exec/exec.go:20]: How can it be required and omitempty?

File: actionners/kubernetes/label/label.go [hunk 3]
```diff
@@ -141,11 +145,21 @@ func Action(action *rules.Action, event *events.Event) (utils.LogLine, error) {
...
-	if len(parameters["labels"].(map[string]interface{})) == 0 {
-		return errors.New("missing parameter 'labels'")
+
+	err = utils.ValidateStruct(config)
+	if err != nil {
+		return err
+	}
+
+	if len(config.Labels) == 0 {
+		return errors.New("parameter 'labels' sould have at least one label.")
```
Comment (Issif) [inline actionners/kubernetes/label/label.go:162]:
```suggestion
		return errors.New("parameter 'labels' should have at least one label.")
```

**Review by Issif — CHANGES_REQUESTED** [review]

**Pull request merged**